modules/ivc_champva/app/controllers/ivc_champva/v1/uploads_controller.rb

```diff
@@ -97,21 +97,15 @@ def get_form_id
         FORM_NUMBER_MAP[form_number_without_colon]
       end
 
-      def build_json(status, error_message)
-        if status.all? { |s| s == 200 }
-          {
-            status: 200
-          }
-        elsif status.all? { |s| s == 400 }
-          {
-            error_message:,
-            status: 400
-          }
+      def build_json(statuses, error_message)
+        unique_statuses = statuses.uniq
+
+        if unique_statuses == [200]
+          { json: {}, status: 200 }
+        elsif unique_statuses.include? 400
```

Comment:
Covers the case where we have all 400s or possibly mixed responses that include a 400. Treats as a 400 and shows error message that was passed in or a generic `Error` string (doing that because in testing the error_message for 400s was `nil`)